.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
All good suggestions.
I'll work out how to do that.
This is my first pull request.

On 13 January 2016 at 06:07, Hong Minhee <notifications@github.com> wrote:

> In .cache/v/cache/lastfailed
> <https://github.com/dahlia/sider/pull/11#discussion_r49549811>:
>
> > @@ -0,0 +1 @@
> > +{}
>
> Could you exclude this file from commit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dahlia/sider/pull/11/files#r49549811>.
>